src/status_im2/contexts/onboarding/welcome/view.cljs

```diff
@@ -32,8 +32,10 @@
     :align-mid?              false
     :page-nav-color          :transparent
     :left-section            {:icon                  :i/arrow-left
+                              ;TODO this is wrong - page nav needs updating
+                              ; should be type:grey, and page nav can use background instead.
                               :icon-background-color colors/white-opa-5
-                              :type                  :shell
```

Comment:
wrong use of button type here, but didn't matter as the background color was being overwritten which defeats purpose of design system approach here.